.github/workflows/sm-cipher.yml

```diff
@@ -0,0 +1,194 @@
+name: SM Cipher Test (1 of 2)
```

Comment:
Why do we need a sm-cipher test in wolfSSL and also in wolfSM? Why duplicated? I recommend only having the one in wolfsm and adding rules to run automatically on schedule. https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule